sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/aio/client.py

```diff
@@ -108,13 +110,15 @@ class CertificateClient(AsyncKeyVaultClientBase):
                                        subject_name="CN=DefaultPolicy",
                                        validity_in_months=12)
 
-        create_certificate_operation = await self._client.create_certificate(
-            vault_base_url=self.vault_url,
-            certificate_name=name,
-            certificate_policy=policy._to_certificate_policy_bundle(),
-            certificate_attributes=attributes,
-            tags=tags,
-            **kwargs
+        create_certificate_operation = CertificateOperation._from_certificate_operation_bundle(
+                await self._client.create_certificate(
```

Comment:
Personally I find nesting makes this hard to read.

Reply:
separated, thanks!